### ISSUE TITLE
@react-email/components@0.0.8

### ISSUE THREAD

**Issue by sysmat** [resendlabs/react-email#997]
### Describe the Bug

npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "workspace:": workspace:*

### Which package is affected (leave empty if unsure)

@react-email/body

### Link to the code that reproduces this issue

aaa

### To Reproduce

- ncu -u
- npm i

### Expected Behavior

to install dependency

### What's your node version? (if relevant)

v21.1.0

**Comment by SamratOS** [comment]
upgrade the pnpm to latest :- npm install -g npm
then install package manager npm install -g pnpm
pnpm install @react-email/body
This will install the @react-email/body package and all of its dependencies
package that is affected by the error message npm ERR! Unsupported URL Type "workspace:": workspace:* is all packages that use the workspace, the following packages:

@react-email/body
@nrwl/react
@nrwl/workspace
lerna
pnpm

**Comment by sysmat** [comment]
1. I have npm 10.2.1
2. I have pnpm 8.10.0
3. I done `pnpm install @react-email/body`
4. I done `ncu -u`
5. I done `pnpm install` but still get error

```
In : "@react-email/body@workspace:*" is in the dependencies but no package named "@react-email/body" is present in the workspace

This error happened while installing the dependencies of @react-email/components@0.0.8
```

**Comment by sysmat** [comment]
- `pnpm create email`  still use old versions `@react-email/components": "0.0.6", "react-email": "1.9.3"`

**Comment by sysmat** [comment]
- even `pnpm add @react-email/body @nrwl/react @nrwl/workspace`
- `ncu -u`  @react-email/components  0.0.7  →  0.0.8
- pnpm install the same error
- with every upgrade this project just break, so you should use semantic versioning for @react-email

**Comment by ccfiel** [comment]
Do you have any updates on this? I am experiencing the same issue as well.

**Comment by bukinoshita** [comment]
Pushing a fix for this

**Comment by sysmat** [comment]
- "@react-email/components": "0.0.9" --> installation OK
-  "@react-email/components": "0.0.9" --> NOT working vercel-invite-user, stripe-welcome, ....
-   "@react-email/components": "0.0.9" a lot of server errors:

```
Body (webpack-internal:///../node_modules/.pnpm/@react-email+body@0.0.4_react@18.2.0/node_modules/@react-email/body/dist/index.mjs:42:5)
Warning: React does not recognize the `pY` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `py` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at a
    at Button (webpack-internal:///../node_modules/.pnpm/@react-email+button@0.0.11_react@18.2.0/node_modules/@react-email/button/dist/index.mjs:130:5)
    at td
    at tr
    at tbody
    at table
    at Section (webpack-internal:///../node_modules/.pnpm/@react-email+section@0.0.10_react@18.2.0/node_modules/@react-email/section/dist/index.mjs:42:5)
    at td
    at tr
    at tbody
```

**Comment by sysmat** [comment]
- "@react-email/components": "0.0.9" a lot of browser errors:

```
hydration-error-info.js:27 Warning: Extra attributes from the server: data-new-gr-c-s-check-loaded,data-gr-ext-installed
    at body
    at html
    at ReactDevOverlay (webpack-internal:///(app-client)/./node_modules/.pnpm/next@13.2.4_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/client/components/react-dev-overlay/internal/ReactDevOverlay.js:61:9)
```

**Comment by sysmat** [comment]
- "@react-email/components": "0.0.9" -> links for template NOT WORKING, if i but direct URL in browser it took 30 sec for response, example: http://localhost:3000/preview/vercel-invite-user
- 0.0.9 version is the total regression
- downgrading to 0.0.7 works fine

**Comment by YewTreeWeb** [comment]
I am currently getting this with `"@react-email/components": "0.0.26"` is there any fix? I'm running Node 22.9

**Comment by gabrielmfern** [comment]
@YewTreeWeb That was because of an issue with the `create-email` script, check your `package.json` and see if you have any unwanted `workspace:´ protocol.
